js/package.json

```diff
@@ -86,7 +86,6 @@
     "gulp-typescript": "5.0.1",
     "ix": "2.5.3",
     "jest": "26.3.0",
-    "jest-environment-node-debug": "2.0.0",
```

Comment:
I didn't see it being used anywhere.

Reply:
yeah I think this used to be necessary for debugging, but has since been fixed in Jest

Reply:
Excellent.